src/test/java/it/pn/frontend/e2e/stepDefinitions/common/BackgroundTest.java

```diff
@@ -151,6 +151,7 @@ public void aggiungiNuovaPECPF() {
 
 
     public void accettazioneDelegaSceltaGruppo(boolean withGroup) {
+        deleghePagoPATest.siSceglieOpzioneAccetta();
```

Comment:
I feel like this can be skipped, because it is already fixed on this PR: https://github.com/pagopa/pn-frontend-e2e/pull/130

Reply:
resolved